deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts

```diff
@@ -232,7 +235,7 @@ export class MigrationConsoleStack extends MigrationServiceCore {
         const environment: { [key: string]: string; } = {
             "MIGRATION_DOMAIN_ENDPOINT": osClusterEndpoint,
             // Temporary fix for source domain endpoint until we move to either alb or migration console yaml configuration
-            "SOURCE_DOMAIN_ENDPOINT": `https://capture-proxy-es.migration.${props.stage}.local:9200`,
+            "SOURCE_DOMAIN_ENDPOINT": sourceClusterEndpoint,
```

Comment:
nit: I dislike that we called this source domain since it very well could not be a Domain, but I realize this is from another change

Reply:
Yeah, that's very true. I think it was done for the symmetry with target domain.